hnn_core/dipole.py

```diff
@@ -138,6 +174,12 @@ def average_dipoles(dpls):
 class Dipole(object):
     """Dipole class.
 
+    An instance of the ``Dipole``-class contains the simulated dipole moment
+    timecourses for L2 and L5 pyramidal cells, as well as their aggregate
+    (``'agg'``). When a ``Dipole`` is created by a call to
+    :func:`~hnn_core.simulate_dipole`, the units of the dipole moment are
+    converted to ``nAm`` (1e-9 Ampere-meters).
```

Comment:
I would actually rephrase this to say that the units are always in ``nAm``. KISS (keep it simple stupid). Whether you get the dipole object from `simulate_dipole` or by instantiating it directly as in `Dipole(...)`